great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -197,6 +170,18 @@ def get_metrics(
         significant dimension) is the number of measurements (e.g., one per Batch of data), while "R^m" is the
         multi-dimensional metric, whose values are being estimated, and details (to be used for metadata purposes).
         """
+        # IDs of Batch objects used to compute the metric -- commonly obtained via the "get_batch_ids()"
+        # method in this module, although it can readily accept the list of Batch IDs generated through any other means.
+        batch_ids: Optional[List[str]] = self.get_batch_ids(
+            domain=domain,
+            variables=variables,
+            parameters=parameters,
+        )
+        if not batch_ids:
```

Comment:
Does this also include an empty list? If we only want to check for nulls, can we use an `is None` check?

Reply:
@cdkini This guards for empty lists as well.